kolibri/plugins/style_guide/assets/src/views/content/inclusive.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+
+  <page-template :completed="true">
+    <h1>Inclusive design</h1>
+
+    <p>
+      At Learning Equality, one of our
+      <a href="https://learningequality.org/about/values/">core values</a> is to
+      "design for everyone". Kolibri should be accessible to the widest possible
+      audience, in the broadest sense of the word 'accessible'. It should run on
+      wide variety of devices, render correctly on a wide variety of screen sizes
+      and assistive technologies, be understandable to people from a wide variety
+      of backgrounds, and be usable by people with a wide variety of abilities and
+      disabilities.
```

Comment:
Maybe change `a wide variety of abilities and disabilities.` for `diverse abilities`.